templates/skeleton/app/entry.server.tsx

```diff
@@ -32,7 +32,8 @@ export default async function handleRequest(
   }
 
   responseHeaders.set('Content-Type', 'text/html');
-  responseHeaders.set('Content-Security-Policy', header);
+  // Todo: bring CSP back
+  // responseHeaders.set('Content-Security-Policy', header);
```

Comment:
not sure how to fix yet. Remix template uses `renderToPipeableStream`, but the doc also said `renderToReadableStream` should be fine